frontend/src/core/utils/fluent/isPluralElement.js

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+
+import { CLDR_PLURALS } from 'core/plural';
+
+import type { PatternElement } from './types';
+
+
+/**
+ * Return true when AST element represents a pluralized string.
+ *
+ * Keys of all variants of such elements are either CLDR plurals or numbers.
+ */
+export default function isPluralElement(element: PatternElement): boolean {
+    if (
+        !(
+            element.type === 'Placeable' &&
+            element.expression && element.expression.type === 'SelectExpression'
+        )
+    ) {
+        return false;
+    }
+
+    return element.expression.variants.every(variant => {
+        return (
+            CLDR_PLURALS.indexOf(variant.key.name) !== -1 ||
+            variant.key.type === 'NumberLiteral'
```

Comment:
I'd revert the order of the two, as you check for `undefined` in the `NumberLiteral` case. Also, doing the type check first, and then accessing the property of the other is better typed, I guess. Flow might not care, but .. ;-)